### ISSUE TITLE
[Important accessibility improvement] Notifications menu should be keyboard accessible

### ISSUE THREAD

**Issue by marcus-herrmann** [owncloud/web#2632]
[This is a ticket for the "accessibility backlog"]

I cannot reproduce and screen shot this issue right now but looking into `Notifications.vue` I recommend the following

- [X] The control that triggers the menu to open should be a button, implemented like this: https://github.com/owncloud/phoenix/pull/2622/files#diff-9cee8f3a250611eb0f0e9d4e356c1853R45
- [X] Usage of the accessible disclosure pattern (See: https://github.com/owncloud/owncloud-design-system/pull/522)
- [X] Usage of correct headline level (`h5` seems off), if using a headline at all
- [ ] Usage of an ordered list for displaying the notification items

**Comment by pascalwengerter** [comment]
Updated the ticket according to our a11y efforts, should be a quick one to finish so adding it to the fastlane

**Comment by pascalwengerter** [comment]
Closing here - the notifications menu has received a throughout rework and hasn't been mentioned in the most recent audit
